django_cloud_deploy/tool_requirements.py

```diff
@@ -177,19 +184,30 @@ def check(cls):
             raise MissingRequirementError(cls.NAME, msg)
 
         try:
-            command = ['docker', 'image', 'ls']
-            subprocess.check_call(command, stdout=subprocess.DEVNULL,
-                                  stderr=subprocess.DEVNULL)
+            if sys.platform.startswith('linux'):
```

Comment:
I don't think that you doc comment is true - you actually check that docker works on Linux . I would structure the code like this:

```python

def _correct_docker_group_setup(self) -> bool:
  user = getpass.getuser()
  p = pexpect.spawn('getent', args)
  try:
    p.expect(user)
  except pexpect.exceptions.EOF:
    return False
  else:
    return True
    
try:
  subprocess.check_call(['docker', 'image', 'ls'])
subprocess.CalledProcessError:
  if sys.platform.statswith('linux') and not self._correct_docker_group_setup():
    msg = ('Docker is installed but you "{user}" are not in the "docker" group, which is
               'required when running docker outside without superuser permissions.'
               'You can run the following commands to fix it: \n'
               '...'
               'For more information, refer to: {link}')
      raise MissingRequirementError(cls.NAME, msg)
    else:
      msg = 'Docker is installed by is not usable. Please follow the installation instructions'
                'at {link}'          
```